README.md

```diff
@@ -106,18 +106,19 @@ The valid values for `ABC_LOG_LEVEL` are `debug`, `info`, `notice`, `warning`,
 ### For `abc templates golden-test`
 
 Usages:
-- `abc templates golden-test record --location=<template_location> <testname>`
-- `abc templates golden-test verify --location=<template_location> <testname>`
+- `abc templates golden-test record --test-name=<test_name> <location>`
```

Comment:
I think --test-name is optional, so the convention would be to put square brackets around it, like `[--test-name=<test_name>]`. Same on the following line.

Reply:
Thank you!